cellprofiler/settings.py

```diff
@@ -722,6 +722,25 @@ def set_value_text(self, value_text):
             self.__default = self.str_to_value(value_text)
         except:
             logger.debug("Number set to illegal value: %s" % value_text)
+            
+    def set_min_value(self, minval):
+        '''Programatically set the minimum value allowed'''
+        self.__minval = minval
+
+    def set_max_value(self, minval):
+        '''Programatically set the maximum value allowed'''
+        self.__maxval = maxval
+        
+    def get_min_value(self):
+        '''The minimum value (inclusive) that can legally be entered'''
+        return self.__minval
+    
+    def get_max_value(self):
+        '''The maximum value (inclusive) that can legally be entered'''
+        return self.__maxval
+    
```

Comment:
Meh, legacy. I had hidden them previously, not seeing any reason why I'd ever want to expose them... and when I did, I figured there would be less chance of code breakage to add accessors than to search and replace all occurrences of the original field. Was my personal choice, going for the path that would risk the least amount of breakage at the expense of style, feel free to change it.